### ISSUE TITLE
CMake uses soname for package version

### ISSUE THREAD

**Issue by jwpleow** [xiph/opus#266]
Hi, 

I notice that for git tag v1.3.1, the so version that is built in linux is 0.8.0 (related issue https://github.com/xiph/opus/issues/114), which is pulled from https://github.com/xiph/opus/blob/bce1f392353d72d77d543bb2069a044ae1045e9d/configure.ac#L25 Should this not be updated to match each other?  This causes issues such as when configuring libopusenc using an installation of opus from git v1.3.1, it complains `Requested 'opus >= 1.1' but version of Opus is 0.8.0`. Is this expected?

**Comment by rillian** [comment]
As with #114, the so version is different from the package version. If you're seeing `0.8.0` in response to an `opus >= 1.1'` configure check, it's finding an incorrectly-configured build of the opus library. That was never a released package version number.

On my machine:
```sh
$ cd opus
$ git checkout v1.3.1
$ ./autogen.sh
[...]
$ ./configure
[...]
$ fgrep Version opus.pc
Version: 1.3.1
```
Make sure the opus.pc or opus-uninstalled.ps your libopusenc build is using has the expected package version number.

**Comment by jwpleow** [comment]
Ah, I installed opus via cmake rather than the autogen.

**Comment by rillian** [comment]
Aha. Yes, it looks like the cmake build is broken here. It's using `opus_library_version` (the soname) where it should be using `opus_package_version`.

@xnorpx are you able to take a look?

**Comment by rillian** [comment]
In fact this is fixed by 6d29f51a4 on the main branch. So this is on us for not having made a release with the cmake fixes since v1.3.1. Sorry for the hassle!

**Comment by jwpleow** [comment]
Oh I see. Thanks for the help!
